### ISSUE TITLE
Base Unet 64 Performance is suddenly worse

### ISSUE THREAD

**Issue by ThodoriKapouranis** [ThodoriKapouranis/Thesis#2]
In March, first Base Unet64 models trained with these MIoU results:

```
Scenario 1,  58.06
Scenario 2,  58.16
Scenario 3,  59.26
```

Now, trained on 15 epochs results in
```
Scenario 1, 39.348
Scenario 2, 31.353
Scenario 3, 46.881
```

Its been a while since training and have introduced 2 new model types, adjusting the base pipeline code to accommodate for these new models and their input/output requirements. 

It might be the case that something has changed from the base scenario that now makes base UNet-64 very poor to predict. 

### TODO
- [ ] Confirm Hyperparameters
- [ ] Confirm learning rate optimizer _**and annealing method**_
- [ ] Confirm epochs? (this info might not be saved, but try training for like 50 epochs)

**Comment by ThodoriKapouranis** [comment]
seems to be fine now
